ajax/libs/rough.js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "rough.js",
+  "title": "rough.js",
+  "filename": "rough.js",
```

Comment:
Let's revise to `rough.min.js` because CDNJS use minified file as filename.
Thank you.